src/grid-layout.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import ContainerSizeContext from './container-type-context';
```

Comment:
@yceballost we need definition on which size (small, mid, wide should be each column on the template, I assumed small from 1 to 5, mid from 6 to 9 and wide just 10 and when not inside a grid layout.

Reply:
we can go with this by the moment and we will review when we can work on it